src/shared/Router.tsx

```diff
@@ -0,0 +1,14 @@
+import { Home } from '@/pages/Home'
+import { BrowserRouter, Routes, Route } from 'react-router-dom'
+
+const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+      </Routes>
```

Comment:
지금은 Router 컴포넌트 안에 라우팅 정보가 모두 하드코딩되어있는데요,
만약 규모가 커지게 되면 이러한 방식으로는 사용하기가 어려울 수 있습니다.

```
// routes.ts
import { Home } from '@/pages/Home'

export const routes = [
  { path: '/', element: <Home /> },
]
```
따라서 라우팅 정보는 위와 같이 별도 상수로 분리하고,

```suggestion
    <Routes>
      {routes.map(({ path, element }) => (
        <Route key={path} path={path} element={element} />
      ))}
    </Routes>
```
위와 같이 해당 데이터를 활용하여 react-router-dom에 적용하는 것도 좋은 방법일 것 같습니다.

다만 현재는 경로가 하나 뿐이라 ㅎㅎ
지금 이와 같은 방식을 적용할 필요는 없구요, 그냥 참고만 해주셔도 될 것 같습니다 :)